auditbeat/module/file_integrity/event.go

```diff
@@ -214,6 +215,17 @@ func NewEvent(
 	return NewEventFromFileInfo(path, info, err, action, source, maxFileSize, hashTypes)
 }
 
+func getDriveLetter(path string) string {
+	volume := filepath.VolumeName(path)
+	if len(volume) == 2 && volume[1] == ':' {
+		letter := path[0]
+		if (letter >= 'a' && letter <= 'z') || (letter >= 'A' && letter <= 'Z') {
```

Comment:
unicode.IsLetter() might be a little clearer.

Reply:
So, Windows drive letters are based off of only ascii letters rather than the full Latin letter set, this line is basically identical to what go does internally for returning volume drives, it's just that they then merge drive letters together with `UNC` paths in the `filepath.VolumeName` call. If it's not clear want me to move it into a helper called `isAsciiLetter`?